src/workloads/tpch/denormalized/1/Q17.yml

```diff
@@ -0,0 +1,21 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-query"
+Description: |
+  Run TPC-H query 17 against the denormalized schema for scale 1.
+
+Clients:
+  Default:
+    QueryOptions:
+      socketTimeoutMS: -1
```

Comment:
I'm curious why do you need to specify `socketTimeoutMS` explicitly for these workloads?

Reply:
I specified it for all workloads- I believe the default timeout is too small for most of the queries. We might be able to remove it for some of them, but it's not necessary since we have another timeout if we don't get a response from a query for 90 minutes.